docs/storybook/src/stories/Button.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./button.css";
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
```

Comment:
Just for my understanding - is this part of the storybook demo, or a component we're actually displaying?

Reply:
yep, this is from the demo. it's not being displayed in the UI, but I sort of like to have it around as I'm learning the features of storybook, it makes for a good reference, particularly the `Button.stories.jsx` file